### PR TITLE
ReSync content French page from English page

### PR DESCRIPTION
**Pull request by niltied**
Hi @hossainemruz,

Yes, it's still me :)... It's a little MR to resync the content of Shortcode french page with your last changes on English page.

### CODE REVIEW

File: content/posts/quickstart/index.fr.md [hunk 1]
```diff
@@ -152,9 +152,9 @@ git push origin HEAD
...
-- Ajouter les informations sur vos compétences en suivant [ce guide](/posts/configuration/sections/skills/).
-- Ajouter les informations sur vos expériences en suivant [ce guide](/posts/configuration/sections/experiences).
-- Déployer votre site sur Github Page en suivant le guide par [ici](/posts/getting-started/github-pages/).
-- Déployer votre site sur Netlify en suivant le guide par [ici](/posts/getting-started/netlify/).
+- Personnaliser l'arrière-plan, le logo, et quelques autres choses de votre site en suivant [ce guide](/posts/configuration/site-parameters/index.fr.md).
+- Ajouter des informations sur vous en suivant [ce guide](/posts/configuration/sections/about/index.fr.md).
+- Ajouter les informations sur vos compétences en suivant [ce guide](/posts/configuration/sections/skills/index.fr.md).
+- Ajouter les informations sur vos expériences en suivant [ce guide](/posts/configuration/sections/experiences/index.fr.md).
+- Déployer votre site sur Github Page en suivant le guide par [ici](/posts/getting-started/github-pages/index.fr.md).
+- Déployer votre site sur Netlify en suivant le guide par [ici](/posts/getting-started/netlify/index.fr.md).
```
Comment (hossainemruz) [inline content/posts/quickstart/index.fr.md:160]: They don't renders as valid link. Please don't specify `index.fr.md` part. Instead add `fr` prefix in the URL. For example:

```
- Personnaliser l'arrière-plan, le logo, et quelques autres choses de votre site en suivant [ce guide](/fr/posts/configuration/site-parameters/).
```

**Review by hossainemruz — CHANGES_REQUESTED** [review]
Fantastic! Thank you for the great work @niltied. Just found a small issue. Can you please fix that? Also, please resolve the conflict.

File: content/posts/quickstart/index.fr.md (hunk 1 above)
Comment (niltied) [reply content/posts/quickstart/index.fr.md]: OK so, it's fixed. But I have a problem with that. When you navigate on this page in french version, if you want go to another page. You will arrive on the target but English version.

Comment (BernatBC) [reply content/posts/quickstart/index.fr.md]: I think the link should be `/fr/posts/configuration/sections/experiences/`.
At least, in my case works.

Comment (niltied) [reply content/posts/quickstart/index.fr.md]: @BernatBC Oh, OK, I will test this solution now. Thank.

Comment (niltied) [reply content/posts/quickstart/index.fr.md]: You're right @BernatBC, it work fine. Thank you so much for your solution.

Comment (niltied) [reply content/posts/quickstart/index.fr.md]: @hossainemruz you can merge, it's OK now.

Comment (hossainemruz) [reply content/posts/quickstart/index.fr.md]: Thank you for fixing the issues.

**Review by hossainemruz — APPROVED** [review]

**Pull request merged**